apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -221,6 +221,16 @@ fields("cluster") ->
             sc(
                 ?R_REF(cluster_k8s),
                 #{}
+            )},
+        {"prevent_overlapping_partitions",
+            sc(
+                boolean(),
+                #{
+                    mapping => "kernel.prevent_overlapping_partitions",
```

Comment:
```suggestion
                    mapping => "vm_args.-kernel prevent_overlapping_partitions",
```
?
(using `dist_net_ticktime` as reference)

Reply:
Yes, this seems more foolproof.